apps/src/blockly/customBlocks/googleBlockly/proceduresBlocks.js

```diff
@@ -146,3 +150,115 @@ GoogleBlockly.Extensions.registerMutator(
   'procedure_def_mutator',
   procedureDefMutator
 );
+
+/**
+ * Constructs the blocks required by the flyout for the procedure category.
+ * Modeled after core Blockly procedures flyout category, but excludes unwanted blocks.
+ * Derived from core Google Blockly:
+ * https://github.com/google/blockly/blob/5a23c84e6ef9c0b2bbd503ad9f58fa86db1232a8/core/procedures.ts#L202-L287
+ * @param {WorkspaceSvg} workspace The workspace containing procedures.
+ * @returns an array of block objects representing the flyout blocks
+ */
+export function flyoutCategory(workspace) {
```

Comment:
what is the purpose of these changes? We did something similar in the modal function editor pr, but put this function inside [functionEditor](https://github.com/code-dot-org/code-dot-org/pull/53114/files#diff-d18875c93b715dba98c6ac0cb7171f9ae07679b1859985ac703a738d8e61625aR318). Should it be moved here?

Reply:
Given https://github.com/code-dot-org/code-dot-org/pull/53154/files#diff-18f81b893f4283e51973795b9e9f866e83c6a4017bb5e3251f16d2384c644be7R151, I assume most if this is duplicate work with your [functionEditor PR](https://github.com/code-dot-org/code-dot-org/pull/53114/files#diff-d18875c93b715dba98c6ac0cb7171f9ae07679b1859985ac703a738d8e61625aR318)?

I'll rebase on staging and test to be confident in refactoring this.